pages/api/roomSession.js

```diff
@@ -0,0 +1,12 @@
+import { newSession } from '../../packages/backend/mongo/connection';
+
+const apiNewSession = async (req, res) => {
+  try {
+    await newSession(req.body.teamName, req.body.roomName);
+    res.status(200);
+  } catch (error) {
+    console.log(error.message);
+  }
+};
+
+export default apiNewSession;

```

Comment:
Why do you call it "session"? and the file is called roomSession and you return apiNewSession also

Reply:
Good point.